operate/qa/integration-tests/src/test/java/io/camunda/operate/metric/ImporterMetricsZeebeImportIT.java

```diff
@@ -190,4 +198,59 @@ public void testBPMNAndDNMModelMetrics() {
                     + Metrics.TAG_KEY_ORGANIZATIONID
                     + "=\"orga\",} 2.0")));
   }
+
+  public static class ManagementPropertyRemoval
```

Comment:
This is required to avoid setting up a server port and a different management port, because Spring Mockmvc would not load the `actuator` endpoints in such case. They are only loaded when there is a single server port configured (and no management port).

Reply:
![giphy](https://github.com/camunda/zeebe/assets/2758593/2e03248e-31a3-4216-848c-30fb6c608c91)